examples/EvaluationDataToApplicationInsights/README.md

```diff
@@ -0,0 +1,77 @@
+# Evaluation Data to Application Insights
+
+This sample shows how to send evaluation data to Application Insights. Evaluation data occurs each time a feature result is determined by the Feature Manager.
+
+## Quickstart
+
+To run this sample, follow these steps:
+
+1. Set the project as the startup project.
+2. Run the project.
+3. Check the Output tab for `Application Insights Telemetry`.
+
+Example Output:
+
+```
+Application Insights Telemetry: {"name":"AppEvents","time":"2023-11-07T19:14:54.3549353Z","tags":{"ai.application.ver":"1.0.0.0"},"data":{"baseType":"EventData","baseData":{"ver":2,"name":"Vote","properties":{"AspNetCoreEnvironment":"Development","DeveloperMode":"true"},"measurements":{"ImageRating":3}}}}
+Application Insights Telemetry: {"name":"AppEvents","time":"2023-11-07T19:14:54.4143414Z","tags":{"ai.application.ver":"1.0.0.0"},"data":{"baseType":"EventData","baseData":{"ver":2,"name":"FeatureEvaluation","properties":{"Label":"A Label","Etag":"An etag","AspNetCoreEnvironment":"Development","DeveloperMode":"true","Variant":"WithColor","FeatureName":"ImageRating","Tags.A":"Tag A value","IsEnabled":"True"}}}}
+```
+
+These logs show what would be emitted to a connected Application Insights resource, even if one is not yet connected.
+
+## Connecting to Application Insights
+
+To flow these events to Application Insights, [setup a new Application Insights resource in Azure](https://learn.microsoft.com/en-us/azure/azure-monitor/app/create-workspace-resource). Once setup, from `Overview` copy the `Connection String` and place it in `appsettings.json` at ApplicationInsights > ConnectionString. After restarting the app, events should now flow to Application Insights.
```

Comment:
We can provide a reference link to teach user how to config the connection string for Application Insights.
I found this link useful: https://learn.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core?tabs=netcorenew%2Cnetcore6#enable-application-insights-server-side-telemetry-no-visual-studio

Reply:
added the link.